scripts/exrrfs_ungrib.sh

```diff
@@ -47,14 +47,17 @@ for fhr in  ${fhr_all}; do
   HHH=$(printf %03d ${fhr})
   GRIBFILE_LOCAL=$(${USHrrfs}/num_to_GRIBFILE.XXX.sh ${knt})
   if [[ "${prefixin}" == "GFS" ]] || [[ "${prefixin}" == "GEFS" ]]; then
-    NAME_FILE=$(echo "${NAME_PATTERNa}" | sed "s/fHHH/f${HHH}/g")
+    #NAME_FILE=$(echo "${NAME_PATTERNa}" | sed "s/fHHH/f${HHH}/g")
```

Comment:
We can remove comments  50, 53, 57 which are no longer needed.
If we want to fine them, we can checkout the commit hash before this PR.

Reply:
removed